cedar-policy/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Starting with version 3.2.4, changes marked with a star (*) are _language breaki
 ## [Unreleased]
 Cedar Language Version: TBD
 
+### Added
+
+- Added experimental `protobufs` code from Brandon Rozek.
```

Comment:
Two comments:
 1. Can we refer to the feature and give a short summary of what it does? It's a substantial feature so I think it'd be worth to extend this item with more information.
 2. Is it common to add the name of the contributor for this kind of contributions?

Reply:
1. Done
2. I had included it because the person merging the code (me) isn't the author, but I've removed it since the information is available on GH and it's kinda weird to include in the changelog.